http/server.go

```diff
@@ -449,14 +449,42 @@ func (s *Server) handleSimpleJSONSearch(ctx *fasthttp.RequestCtx) {
 }
 
 func (s *Server) handleSimpleJSON(ctx *fasthttp.RequestCtx, method string) {
-	req, err := simpleJSONRequestFactory(method, ctx.PostBody())
+	requests, err := simpleJSONRequestFactory(method, ctx.PostBody())
 	if err != nil {
 		s.logger.ErrorWith("can't initialize simplejson request", "error", err)
 		ctx.Error(fmt.Sprintf("bad request - %s", err), http.StatusBadRequest)
 		return
 	}
-	// passing session in order to easily pass token, when implemented
+
+	var results interface{}
+	for _, req := range requests {
+		result, err := s.executeSimpleJSONSubRequest(req, ctx)
+		if err != nil {
+			ctx.Error(fmt.Sprintf("Error querying: %s", err), http.StatusInternalServerError)
+			return
+		}
+
+		results = appendSimpleJSONResults(results, result)
+	}
+
+	_ = s.replyJSON(ctx, results)
+}
+
+func appendSimpleJSONResults(origin, resToAdd interface{}) interface{} {
+	if origin == nil {
+		return resToAdd
+	}
+	switch origin.(type) {
+	case []timeSeriesOutput:
+		return append(origin.([]timeSeriesOutput), resToAdd.([]timeSeriesOutput)...)
+	default:
+		return append(origin.([]tableOutput), resToAdd.([]tableOutput)...)
```

Comment:
Unsafe 💥🚗

Reply:
Safe enough for us, these types and all are internal.